src/components/Dashboard/Overview/Overview.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { Grid } from "semantic-ui-react";
+import { Logo, Header, Popup } from "decentraland-ui";
+import { FormattedMessage, FormattedPlural, FormattedNumber, useIntl } from "react-intl";
+import { copyToClipboard, getMonthDiff } from "../../../utils";
+import ManaWidget from "../../ManaWidget";
+import useResponsive from "../../../hooks/useResponsive";
+import Responsive from "semantic-ui-react/dist/commonjs/addons/Responsive";
+import Copy from "../../../images/copy.svg";
+import Link from "../../../images/link.svg";
+import DaiLogo from "../../../images/dai_logo.svg";
+import UsdtLogo from "../../../images/usdt_logo.svg";
+import UsdcLogo from "../../../images/usdc_logo.svg";
+
+import "./Overview.css";
+import useReviewUrl from "../../../hooks/useReviewUrl";
+
+export default function Overview(props) {
+  const { address, contract } = props;
+
+  const { symbol, released, balance, start, cliff, duration } = contract;
+  const total = balance + released;
+  const vestingMonths = getMonthDiff(start, start + duration);
+  const vestingCliff = getMonthDiff(start, cliff);
+
+  const logo = {
```

Comment:
You can put this object outside the function component.

Reply:
Done!